schema.graphql

```diff
@@ -163,3 +158,25 @@ type IndexerReward @entity {
   additions: BigInt!
   removals: BigInt!
 }
+
+type Offer @entity {
+  id: ID! # offerID
+  consumer: String!
+  deployment: Deployment!
+  planTemplate: PlanTemplate!
```

Comment:
Is this mean to be `planTemplateId`?

Reply:
No. It is supposed to connect `PlanTemplate` entity.